records/validation_test.go

```diff
@@ -35,15 +37,17 @@ func TestValidateResolvers(t *testing.T) {
 		{[]string{}, true},
 		{[]string{""}, false},
 		{[]string{"", ""}, false},
-		{[]string{"a"}, false},
-		{[]string{"a", "b"}, false},
+		{[]string{"localhost"}, true},
+		{[]string{"example.com"}, true},
+		{[]string{"example.com", "localhost"}, true},
+		{[]string{"localhost", "localhost"}, false},
 		{[]string{"1.2.3.4"}, true},
-		{[]string{"1.2.3.4.5"}, false},
+//		{[]string{"1.2.3.4.5"}, false},
```

Comment:
How do you validate both 1.2.3.4.5 AND hostnames? I don't know of any way to do it.